### PR TITLE
Adds AccountOverlay mobile component

### PR DESCRIPTION
**Pull request by sahilmob**

### CODE REVIEW

**Review by goldyydev — CHANGES_REQUESTED** [review]
If you go on [this](https://www.figma.com/proto/CS01VVLR7ArQl0afYFkNj3/Web-App?node-id=4300%3A65&scaling=min-zoom) mode you can see that the AccountOverlay is supposed to be a **draggable** overlay (with the handle at the top). you have made the div itself but did not make it draggable nor is it currently an overlay.

I'd suggest you use something like react-draggable for this.

**Comment by sahilmob** [comment]
Yes it was clear for me that it is draggable, however, I focused only on the UI part of the component.
Anyway I will make draggable and update the PR.

**Review by goldyydev — CHANGES_REQUESTED** [review]
You took probably the most annoying component out there. hopefully this is the last fix 🤞 

image

**Comment by sahilmob** [comment]
@goldyydev No problem at all :smile:

**Review by HarrisonMayotte — APPROVED** [review]

File: kibbeh/src/ui/mobile/AccountOverlay.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+export const AccountOverlay: React.FC<AccountOverlyProps> = ({
+  children,
+  position,
+  bounds,
+  dragHandler,
+  closeHandler,
+  className = "",
+  axis = "both",
```
Comment (goldyydev) [inline kibbeh/src/ui/mobile/AccountOverlay.tsx]: Just a minor fix, this wouldn't be used for any xy dragging.
```suggestion
  axis = "y",
```

**Review by goldyydev — CHANGES_REQUESTED** [review]
Just a minor fix

**Review by goldyydev — APPROVED** [review]

**Pull request merged**